### PR TITLE
Add a `0.02%` threshold to Codecov project

### PR DESCRIPTION
**Pull request by antalszava**
**Context and change**
Update the accepted project % to `0.02%`, so that PRs that remove more source lines than they add will no longer fail the Codecov project check.

**Posibble drawback**

The `0.02%` tolerance is a double-edged sword: it may allow false positives and so PR authors and reviewers should be careful and check if a decreased coverage percentage means uncovered lines.

### CODE REVIEW

File: pennylane/operation.py [hunk 1]
```diff
@@ -110,20 +110,6 @@
...
-        obj = getattr(qml.measurements, name)
-        warning_string = f"qml.operation.{name} is deprecated. Please import from qml.measurements.{name} instead"
-        warnings.warn(warning_string, UserWarning)
-        return obj
-    try:
-        return globals()[name]
-    except KeyError as e:
-        raise AttributeError from e
-
-
```
Comment (Jaybsoni) [inline pennylane/operation.py:126]: Why was this removed ..?

Comment (antalszava) [reply pennylane/operation.py:126]: To remove some lines so that potentially codecov/project would fail.

Comment (antalszava) [reply pennylane/operation.py:126]: Also, we would like to remove these lines (and tests) for v0.24.0 as part of the deprecation cycle of the import path change.

**Review by timmysilv — APPROVED** [review]
